client/www/components/dash/Auth.tsx

```diff
@@ -164,13 +171,23 @@ export default function Auth(props: { emailOnly?: boolean; info?: ReactNode }) {
       isLoading: true,
     }));
 
-    verifyMagicCode({ email, code }).catch((err) => {
-      setState((prev) => ({
-        ...prev,
-        isLoading: false,
-        error: errorFromVerifyMagicCode(err),
-      }));
-    });
+    verifyMagicCode({ email, code }).then(
+      ({ token }) => {
+        const ticket = props.ticket;
+        if (!ticket) return;
+
+        claimTicket({ ticket, token }).then(() => {

```

Comment:
I notice we have the `claimTicket` already in the Dash Root. Is it necessary to call it here?

My thinking is, it should only be called there, as it doesn't matter from 'where' the ticket comes